spec/spec_helper.rb

```diff
@@ -1,8 +1,42 @@
+ENV['RACK_ENV'] = 'test'
+
+require File.join(File.dirname(__FILE__), '..', 'app/app.rb')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.